app/utils/deviceInfo.js

```diff
@@ -9,3 +9,13 @@ export const isAndroid = !isIOS;
 export const getReadableVersion = DeviceInfo.getReadableVersion();
 export const getBundleId = DeviceInfo.getBundleId();
 export const getDeviceModel = DeviceInfo.getModel();
+
+let _width = null;
+export const setWidth = width => _width = width;
+
+export const isTablet = (withWidth = true) => {
+	if (withWidth) {
+		return DeviceInfo.isTablet() && _width > 700;
```

Comment:
It didn't work because I make a mistake.. sorry, try again with new commits..
700 is a best minimum width to work with a sidebar with width of 320, I followed proportions of native iOS client.